src/app/components/error/index.html

```diff
@@ -1,13 +1,196 @@
+            </div>
+          </sky-docs-visual-example-image>
+                
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+          <p>Don’t use the security error type when a user doesn’t have permission to view a portion of the page such as a tile on a record page. Hide the tile from view instead.</p>
+        </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div>
+              <img src="~/assets/img/guidelines/error/error-usage-5.png" />
+                <div class="caption caption-danger sky-text-danger">Don't use a security error in a portion of a page or modal. Hide the restricted content instead.</div>
+            </div>
+          </sky-docs-visual-example-image>
+                
+        </sky-docs-visual-example>
+        
+    </sky-demo-page-content>
+        
+    <sky-demo-page-content sectionHeading="Anatomy">
+        
+        <sky-docs-anatomy imagePath="~/assets/img/guidelines/error/error-anatomy.png">
+          <sky-docs-anatomy-item anatomyNumber="1">Title</sky-docs-anatomy-item>
+            <sky-docs-anatomy-item anatomyNumber="2" isOptional="true">Image</sky-docs-anatomy-item>
+            <sky-docs-anatomy-item anatomyNumber="3" isOptional="true">Description</sky-docs-anatomy-item>
+            <sky-docs-anatomy-item anatomyNumber="4" isOptional="true">Action</sky-docs-anatomy-item>
+          </sky-docs-anatomy>
+    
+    </sky-demo-page-content>
+        
+    <sky-demo-page-content sectionHeading="Options">
+        
+        <h3>Error type</h3>
+        
+        <sky-docs-visual-example>
+            <sky-docs-visual-example-text>
+              <p>Use the broken error type in situations where the underlying content doesn’t load or when there is an issue reaching the server, such as 500 errors.</p>
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-option-broken.png" />
+                </div>
+            </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+            <sky-docs-visual-example-text>
+              <p>Use the construction error type when a page is under maintenance or is experiencing some sort of known outage.</p>
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-option-construction.png" />
+                </div>
+            </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+            <sky-docs-visual-example-text>
+              <p>Use the not found error type when there is an error loading a page due to a 404 error.</p>
```

Comment:
I'd change the when clause to "when a page fails to load because of a 404 error."

Also, if we use code snippets for the error types, then `notfound` should be one word. But if we stick with generic references, it's fine as is.